packages/opencensus-resource-util/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "pretty": true,
+    "module": "commonjs",
+    "target": "es6",
+    "strictNullChecks": false,
```

Comment:
Why set `strictNullChecks` to `false`? I would think `true` would give us stronger type guarantees and more safety as the new library is developed.

Reply:
SGTM, done.